### PR TITLE
feat: add create draft

### LINKED ISSUE #27: Create an Email draft using Node
Code sample to create an email draft using Nylas Node SDK.

### PR DESCRIPTION
**Pull request by MoskalykA**
## PR Acceptance Criteria
Closes #27 

## Guidelines
* [x] PR follows the [Contributing guidelines](https://github.com/nylas-samples/nylas-hacktoberfest-2022/blob/main/CONTRIBUTION.md)?
* [x] PR Adheres to our [Code of Conduct](https://github.com/nylas-samples/nylas-hacktoberfest-2022/blob/main/CODE_OF_CONDUCT.md)?
* [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
* [x] Ensure you are assigned the issue before creating a PR.
* [x] Only one issue per PR is allowed. This simplifies the code review.

## Code complete
* [x] A `README.md` available describing 1) how to setup, 2) how to test the code and 3) the expected output? Here is an [example](code-samples/account/node/read-account/readme.md). 
* [x] The code sample in the correct location (i.e. `code-sample/email/node/read-email`)?
* [x] Link added to the repository readme (view `# Code Snippets created by the Nylas Community` with examples) describing what the code sample does for others to try out
* [x] The code can be run using Github Codespaces, so the maintainer can try out the code sample remotely to ensure everything is working before we accept the PR.

## For Maintainer Use Only (does not apply to PR Author)
* [x] Purpose of code sample is clear
* [x] The code sample is readable
* [x] Steps try out the code are provided
* [x] Code sample runs on codespaces environment
* [x] Code works as expected
* [x] PR label `hacktoberfest-accepted` added

### CODE REVIEW

File: code-samples/email/node/create-draft/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const Nylas = require('nylas')
+const { default: Draft } = require('nylas/lib/models/draft')
+
+Nylas.config({
+   clientId: process.env.CLIENT_ID,
+   clientSecret: process.env.CLIENT_SECRET
```
Comment (relaxedtomato) [inline code-samples/email/node/create-draft/index.js:6]: have you tested this locally - how do we access these values, do we need to have `dotenv`?

**Review by relaxedtomato — CHANGES_REQUESTED** [review]

**Comment by MoskalykA** [comment]
I had forgotten about the famous dotenv

File: code-samples/email/node/create-draft/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+})
+
+const nylas = Nylas.with(process.env.ACCESS_TOKEN)
+new Draft(nylas, {
+  subject: 'Creating a draft',
+  body: 'This is where a draft is created',
+  to: [{ name: 'You', email: 'you@nylas.com' }]
+})
```
Comment (relaxedtomato) [inline code-samples/email/node/create-draft/index.js:16]: This is really close, one thing to note is that the draft is not available in my email unless I add `draft.save();`

Can you consider adding this after creating the draft?

File: code-samples/email/node/create-draft/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+npm install
+node index.js
+```
+
+But if you use yarn:
+```bash
+cd code-samples/email/node
+yarn
+node index.js
+```
```
Comment (relaxedtomato) [inline code-samples/email/node/create-draft/readme.md:27]: Can we add a `#Usage` section to describe what the desired output is?

Even if its receiving a draft email, that's sufficient to mention with an image:
Screen Shot 2022-10-13 at 3 33 15 PM

**Review by relaxedtomato — CHANGES_REQUESTED** [review]
Minor changes, but should be good to go afterwards.

**Comment by MoskalykA** [comment]
Is it good?

File: code-samples/email/node/create-draft/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+const nylas = Nylas.with(process.env.ACCESS_TOKEN)
+const draft = new Draft(nylas, {
+  subject: 'Creating a draft',
+  body: 'This is where a draft is created',
+  to: [{ name: 'You', email: 'you@nylas.com' }]
+})
+
+draft.save()
```
Comment (relaxedtomato) [inline code-samples/email/node/create-draft/index.js:18]: ⚡

**Review by relaxedtomato — APPROVED** [review]

**Comment by relaxedtomato** [comment]
@MoskalykA thanks for your contribution for Hacktoberfest, consider picking up [another ticket](https://github.com/nylas-samples/nylas-hacktoberfest-2022/issues) or by [suggesting a code sample you like to share](https://github.com/nylas-samples/nylas-hacktoberfest-2022/issues/new/choose)!

**Pull request merged**